es-core/src/resources/Font.cpp

```diff
@@ -600,12 +600,12 @@ float Font::getNewlineStartOffset(const std::string& text, const unsigned int& c
 			return 0;
 		case ALIGN_CENTER:
 			{
-				unsigned int endChar = (unsigned int)text.find('\n', charStart);
+				size_t endChar = (unsigned int)text.find('\n', charStart);
```

Comment:
Changed unsigned int to size_t.

Reply:
What lead us to this change? The cast is explicitly (unsigned int), so the cast and the data type was clear.

Reply:
It's actually the line below which causes the warning where endChar is compared to std::string::npos which is of type size_t.